extensions/cba_settings/cba_settings.cpp

```diff
@@ -0,0 +1,335 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include "trim.h"
+
+#define VERSION "1.0.0"
+
+#define RET \
+strncpy_s(output, outputSize, result.c_str(), _TRUNCATE);\
+return
+
+#define RETURN(val)\
+strncpy_s(output, outputSize, result.c_str(), _TRUNCATE);\
+return (val)
+
+#define SYNTAX_ERROR_WRONG_PARAMS_SIZE 101
+#define SYNTAX_ERROR_WRONG_PARAMS_TYPE 102
+#define PARAMS_ERROR_TOO_MANY_ARGS 201
+#define EXECUTION_WARNING_TAKES_TOO_LONG 301
+
+#define TEMP_FILE "userconfig/temp"
+#define SETTINGS_FILE "userconfig/cba_settings.sqf"
+#define DELIM ":"
+#define MAX_CALLBACK_LENGTH 10000
+
+using namespace std;
+
+string version() {
+    return VERSION;
+}
+
+string clear() {
+    ofstream file;
+    file.open(SETTINGS_FILE, ios::out);
+    file.close();
+    return "true";
+}
+
+string load(int strlen) {
```

Comment:
strlen is the name of a function, please don't use that as variable name.

Reply:
Haha.